samples/meetings-content-bubble/csharp/ContentBubble/Bots/ContentBubbleBot.cs

```diff
@@ -23,8 +32,9 @@ public class ContentBubbleBot : TeamsActivityHandler
     {
         private readonly IConfiguration _config;
         private readonly MeetingAgenda _agenda;
+        private IHttpClientFactory _httpClientFactory;
 
-        public ContentBubbleBot(IConfiguration configuration)
+        public ContentBubbleBot(IConfiguration configuration, IHttpClientFactory httpClientFactory)
```

Comment:
Let's update the name of the bot/file to be TargetedInMeetingNotifiationBot :) We are intentionally moving away from "ContentBubble"

Reply:
Yes, updated the file name